JitsiConference.js

```diff
@@ -151,7 +151,8 @@ export default function JitsiConference(options) {
      */
     this.deferredStartP2PTask = null;
 
-    const delay = parseInt(options.config.backToP2PDelay, 10);
+    const delay
+        = parseInt(options.config.p2p && options.config.p2p.backToP2PDelay, 10);
```

Comment:
I wish there was a nicer way to access these properties, but I can't think of any which would be simple so let it be...